soccer/src/soccer/soccer_mom/soccer_mom.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <rclcpp/rclcpp.hpp>
+#include <rj_constants/topic_names.hpp>
+
+#include "std_msgs/msg/string.hpp"
+#include "rj_msgs/msg/team_color.hpp"
+
+namespace tutorial {
+
+class SoccerMom : public rclcpp::Node {
+public:
+    SoccerMom();
+
+private:
+    void publish_fruit(bool is_blue_team);
+    rclcpp::Publisher<std_msgs::msg::String>::SharedPtr mom_msg_pub_;
+    rclcpp::Subscription<rj_msgs::msg::TeamColor>::SharedPtr team_color_sub_;
+    const std::string pub_topic_ = "team_fruit";
```

Comment:
In the future, constants are better off like this:
```suggestion
    static constexpr std::string_view pub_topic_ = "team_fruit";
```
or maybe...
```suggestion
    static constexpr char[] pub_topic_ = "team_fruit";
```
Or, better still, put it in one of the files that contains other topic constants, since with a real topic, more than one node ought to use it, or there is no real point of having it.